docs/actions/match.md

```diff
@@ -161,6 +161,23 @@ end
 
 Then all your team has to do is `fastlane certificates` and keys, certs and profiles for all targets will be synced.
 
+#### Handle multiple apps per developer/distribution certificate
+
+If you want to use a single developer and distribution certificate for multiple apps belonging to the same development team, you may use the same signing identities repository and branch to store the signing identities for your apps:
```

Comment:
... developer and distribution ...: This should be "or" probably?

Reply:
@janpio both "or" and "and" are valid in this case since the user can have a single developer certificate, a single distribution certificate which can be used for multiple apps. Each app, in this case, will become individual provisioning profiles based on certificates from above.

Reply:
Yep, that is the perfect solution. Actually makes it clearer what is possible! 👍